### ISSUE TITLE
Caching downloaded JS scripts, CSS files & fonts.

### ISSUE THREAD

**Issue by BinarySoftware** [enso-org/ide#957]
### Summary
User wants to work offline on their project, and redownloading scripts everytime IDE opens seems ineffective.

**Comment by MichaelMauderer** [comment]
Ideally for the electron app they should not just be cached, but packaged with the app.

**Comment by sylwiabr** [comment]
is it the same as https://app.zenhub.com/workspaces/enso-ide-5e99c2847b8a3027b334f1b9/issues/enso-org/ide/809 ?

<blockquote><img src="No image" width="48" align="right"><div><img src="/dist/favicon/favicon-16x16.png" height="14"> ZenHub</div><div><strong><a href="https://zenhub.com">ZenHub</a></strong></div><div>Project Management and Software Development with GitHub | ZenHub</div></blockquote>

**Comment by etlweather** [comment]
This also prevents the fully off-line use of this program - as in, a computer with no internet access as the program tries to download some config.json file from GitHub at startup.

**Comment by wdanilo** [comment]
@etlweather that's right. We want to improve it. How big trouble does it cause on your end currently?

**Comment by etlweather** [comment]
@wdanilo It makes it impossible to use it at all since my system is totally disconnected from the internet. The software looks promising but couldn't try it unfortunately.

**Comment by etlweather** [comment]
Just checking if there was any update on this, I was really interested by this platform.
